### DISCUSSION TITLE
javascript

### DISCUSSION THREAD

**Discussion by M1S1A2M8** [community/community#123950] [npm]
how to turn javascript on and off

**Comment by davevad93** [comment]
Hi @M1S1A2M8 ,

Here's how to enable or disable JavaScript in your web browser:

- Chrome
  1. Go to `Settings`.
  2. Go to `Privacy and Security` section and select `Site Setting`.
  3. Click on `JavaScript` in the `Content` group.
  4. Select `Sites can use JavaScript` to enable it or `Don't allow sites to use JavaScript` to disable it.

- Firefox
  1. Enter `about:config` in the address bar.
  2. Click on `Accept the Risk and Continue`.
  3. Enter `javascript.enabled` in the configuration search bar.
  4. Click on `toggle` to change its value from `true` (enabled) to `false` (disabled).

- Microsoft Edge
  1. Go to `Settings`.
  2. Click on `Cookies and Site Permissions` and scroll down to the `All Permissions` section. 
  3. Select `JavaScript`.
  4. Click on the toggle switch in the `Allowed (recommended)` section to disable or enable it.
